homeassistant/components/enocean/sensor.py

```diff
@@ -281,3 +292,21 @@ def value_changed(self, packet):
             self._attr_native_value = "tilt"
 
         self.schedule_update_ha_state()
+
+class EnOceanopening(EnOceanSensor):
```

Comment:
```suggestion


class EnOceanOpening(EnOceanSensor):
```

Reply:
Thanks for your suggestion - 
Ok with me :-)